package.json

```diff
@@ -15,6 +15,7 @@
     "start": "node src/app.js",
     "lint": "eslint .",
     "test": "istanbul test -- _mocha",
+    "test-oracle": "LEDGER_DB_ENV=oracledev DYLD_LIBRARY_PATH=/opt/oracle/instantclient node node_modules/.bin/istanbul test -- _mocha",
```

Comment:
So the `npm run test-oracle` command fails with
```
Knex: run
$ npm install strong-oracle --save
Error: dlopen(/Users/matt/ripple_forks/five-bells-ledger/node_modules/strong-oracle/build/Release/oracle_bindings.node, 1): Library not loaded: /ade/b/3071542110/oracle/rdbms/lib/libocci.dylib.11.1
```
Unless the full path to `istanbul` is referenced in the package.json. Perhaps @naoitoi can shed some light on the issue.